layouts/page/pricing.html

```diff
@@ -926,6 +926,21 @@ <h2 class="text-3xl text-center mt-10 mb-0 lg:hidden">Pulumi Service</h2>
                 </label>
             </li>
 
+            <li class="faq-item text-2xl">
+                <label class="cursor-pointer w-full">
+                    <input type="checkbox" class="absolute invisible"></input>
+                    <div class="faq-item-label">
+                        <p>Is Pulumi SOC 2 compliant?</p>
+                        <p class="flex-grow text-right"><i class="fas fa-chevron-down"></i></p>
+                    </div>
+                    <div class="faq-item-text hidden text-base">
+                            <p>
+                                Yes, Pulumi is SOC 2 certified. Pulumi is committed to operational excellence for our customers.
```

Comment:
```suggestion
                                Yes, Pulumi has completed the SOC 2 Type 2 compliance process. Pulumi is committed to operational excellence for our customers.
```

Reply:
we're not supposed to use the word "certified" ... FYI only ... even though this is the phrase that everyone commonly uses!  :)